core/tests/test_apps.cardano.address.py

```diff
@@ -406,13 +426,26 @@ def test_shelley_address_with_byron_namespace(self):
         keychain = Keychain(node)
 
         with self.assertRaises(wire.DataError):
-            derive_base_address(keychain, [44 | HARDENED, 1815 | HARDENED, HARDENED, 0, 0], 0, None)
+            address_parameters = CardanoAddressParametersType(
+                address_type=CardanoAddressType.BASE_ADDRESS,
+                address_n=[44 | HARDENED, 1815 | HARDENED, 0 | HARDENED, 0, 0]
+            )
+            derive_address(keychain, address_parameters, 0)
 
         with self.assertRaises(wire.DataError):
-            derive_pointer_address(keychain, [44 | HARDENED, 1815 | HARDENED, HARDENED, 0, 0], 0, CardanoCertificatePointerType(0, 0, 0))
+            address_parameters = CardanoAddressParametersType(
+                address_type=CardanoAddressType.POINTER_ADDRESS,
+                address_n=[44 | HARDENED, 1815 | HARDENED, 0 | HARDENED, 0, 0],
+                certificate_pointer=CardanoCertificatePointerType(0, 0, 0)
```

Comment:
After a discussion with ppershing, we used BlockchainPointer in Ledger, so perhaps let's do so here, too. It is too general in a sense (it does point to a certificate, not anything else in blockchain), but much more clearly reflects that the pointer is nothing related to our code, but to some external reality. (Another PR.)

Reply:
Added to my todo list.